plugins/modules/portage.py

```diff
@@ -333,10 +341,11 @@ def emerge_packages(module, packages):
     """Run emerge command against given list of atoms."""
     p = module.params
 
-    if p["noreplace"] and not p["changed_use"] and not p["newuse"] and not (p["update"] or p["state"] == "latest"):
+    if p["noreplace"] and not p["changed_deps"] and not p["changed_use"] and not p["newuse"] and not (p["update"] or p["state"] == "latest"):
```

Comment:
This line likely would change when the code is reformatted. That's why the sanity tests fail.

(You probably need to rebase to current `main` to run the formatter, since we only added it to the repository yesterday.)

Reply:
I've rebased against main now, will check the tests yet again :-)